src/hooks/use-commerce-machine.ts

```diff
@@ -32,11 +32,19 @@ const extractPricesFromPricingData = (pricingData: PricingData): Prices => {
   return pickBy({annualPrice, quarterlyPrice, monthlyPrice})
 }
 
-export const useCommerceMachine = () => {
+type Options = {
+  initialPlan: 'annualPrice' | 'monthlyPrice' | 'quarterlyPrice'
+}
+
+export const useCommerceMachine = (
+  options: Options = {initialPlan: 'annualPrice'},
+) => {
```

Comment:
This makes the initial plan selected by the Commerce Machine hook configurable. It defaults to annual, but for this CTA overlay, we can override it as monthly.